trakt/tv.py

```diff
@@ -659,6 +665,37 @@ def scrobble(self, progress, app_version, app_date):
         """
         return Scrobbler(self, progress, app_version, app_date)
 
+    def checkin(self, app_version, app_date, message="", sharing={},
```

Comment:
same here, `sharing` should default to `None`

Reply:
done